rocketpool/api/network/stats.go

```diff
@@ -128,7 +128,7 @@ func getStats(c *cli.Context) (*api.NetworkStatsResponse, error) {
 	wg.Go(func() error {
 		multicallerAddress := common.HexToAddress(cfg.Smartnode.GetMulticallAddress())
 		balanceBatcherAddress := common.HexToAddress(cfg.Smartnode.GetBalanceBatcherAddress())
-		contracts, err := rpstate.NewNetworkContracts(rp, multicallerAddress, balanceBatcherAddress, false, nil)
```

Comment:
cool- great example of why it's good to update the library to remove the argument first. we missed this call earlier.